### DISCUSSION TITLE
What does skip schema validation means in the sync sidebar

### DISCUSSION THREAD

**Discussion by wvides** [argoproj/argo-cd#7143] [Q&A] [answered]
Hello I've been looking around for documentation on what the **skip schema validation** checkbox does and I cannot find anything that states what ArgoCD will not do in case this is enabled on the sync operation or what validation it would be doing normally.

Is there any diagram documentation with the lifecycle of what ArgoCD  would do in a normal sync operation?, the code section where I can see that would do as well

Thank you!

**Comment by chetan-rns** [comment, marked as answer]
Hi @wvides, I think selecting `Skip Schema Validation` will set `Validate=false` in the sync options for the app. It informs Argo CD to use the `--validate=false` flag when running kubectl apply

https://argo-cd.readthedocs.io/en/latest/user-guide/sync-options/#disable-kubectl-validation
